tests/unit/services/emailOtp.test.ts

```diff
@@ -0,0 +1,59 @@
+import { AxiosInstance } from 'axios';
+import { BaseError } from '../../../src/errors';
+import { EmailCodeSendReq, EmailCodeValidateReq } from '../../../src/generated';
+import { EmailOTP } from '../../../src/services';
+import Utils from '../../utils';
+
+describe('EmailOtp class', () => {
+  let axiosInstance: AxiosInstance;
+
+  beforeEach(() => {
+    axiosInstance = Utils.AxiosInstance();
+  });
+
+  it('should create EmailOTP instance with AxiosInstance', () => {
+    const emailOTP = new EmailOTP(axiosInstance);
```

Comment:
I see that we distinguish between unit and integration tests. Currently, the unit tests are also depending on out Corbado backend (didn't check all of them but at least emailOtp.test.ts does). From my experience unit tests are independent from external systems like databases or other APIs. I think we should either mock the API in these unittests or remove them, otherwise they look more like integration tests to me, what do you think?

Reply:
Yes, you are right. Ideally, unit tests should be completely isolated. I will note this down as a priority issue